paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/FakeBacsMandateConfirmationLauncher.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.bacs
+
+import com.stripe.android.paymentsheet.PaymentSheet
+
+internal class FakeBacsMandateConfirmationLauncher : BacsMandateConfirmationLauncher {
+    override fun launch(
+        data: BacsMandateData,
+        appearance: PaymentSheet.Appearance?
+    ) {
+        // No-op
```

Comment:
Do we want to make assertions that this was called with specific data?

Reply:
I did do it in the tests with `spy`. Do we want to avoid using `Mockito` for this and add some argument inputs in the fake launcher and fake launcher factory?